Soccar [UI]/src/nl/soccar/ui/fx/drawable/BoostMeterUiFx.java

```diff
@@ -0,0 +1,42 @@
+package nl.soccar.ui.fx.drawable;
+
+import javafx.scene.canvas.GraphicsContext;
+import nl.soccar.library.Car;
+import nl.soccar.ui.DisplayConstants;
+import nl.soccar.ui.drawable.Drawable;
+import nl.soccar.ui.drawable.GameCanvas;
+import nl.soccar.ui.util.PhysicsUtilities;
+
+/**
+ * A NotificationUiFx object represents a JavaFX Drawable of a Notification.
+ * It keeps track of the Notification and NotificationPhysics models and provides methods to draw and
+ * update the models.
+ *
+ * @author PTS34A
+ */
+public class BoostMeterUiFx extends Drawable<Car> {
+
+    /**
+     * Initiates a new NotificationUiFx Object using the given parameters.
+     *
+     * @param canvas The canvas on which this Notification is placed.
+     */
+    public BoostMeterUiFx(GameCanvas canvas, Car car) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): car [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)